Cargo.toml

```diff
@@ -74,7 +78,7 @@ swc_ecma_transforms_base = {version = "0.31.1", path = "./ecmascript/transforms/
 swc_ecma_utils = {version = "0.44.2", path = "./ecmascript/utils"}
 swc_ecma_visit = {version = "0.38.1", path = "./ecmascript/visit"}
 swc_ecmascript = {version = "0.63.1", path = "./ecmascript"}
-swc_node_base = {version = "0.3.0", path = "./node/base"}
+swc_node_base = {version = "0.3.0", path = "./node/base", optional = true}
```

Comment:
Please move this to `dev-dependencies`. It's required to get correct profiling result.

Reply:
Also, please make it not optional.